lib/actions/projectOverviewAction.js

```diff
@@ -0,0 +1,16 @@
+const ProjectOverviewResponse = require('./projectOverviewResponse.js');
+const ProjectLink = require('../projectLink');
+const MySQLDriver = require('../dbDriver/mysql');
+
+class ProjectOverviewAction {
+  // eslint-disable-next-line class-methods-use-this
+  async execute(params, context) {
+    // eslint-disable-next-line new-cap
+    const projectLink = new ProjectLink(context.project);
```

Comment:
An instance of a `ProjectLink` should always represent a fully formed `ProjectLInk`. That is, it should have a Project, Label, and Url. Here we are constructing single `ProjectLink` for the orthogonal purpose of retrieving other  instances. I would expect a static method here.